src/main/java/com/hillel/items_exchange/controller/UserController.java

```diff
@@ -53,4 +63,53 @@
         }
         return new ResponseEntity<>(userService.update(userDto, user), HttpStatus.ACCEPTED);
     }
+
+    @GetMapping("/child")
+    public ResponseEntity<List<ChildDto>> getChildren(Principal principal) {
+        return new ResponseEntity<>(userService.getChildren(getUser(principal.getName())), HttpStatus.OK);
+    }
+
+    @PostMapping("/child")
+    public ResponseEntity<HttpStatus> addChildren(@RequestBody @Size(min = 1, message = "{exception.invalid.dto}") List<@Valid ChildDto> childrenDtoList,
```

Comment:
Don't display all parameters in one line it's really bad approach. You could do it in such way:
```
    public ResponseEntity<HttpStatus> addChildren(@RequestBody @Size(min = 1, message = "{exception.invalid.dto}")
                                                          List<@Valid ChildDto> childrenDtoList,
                                                  Principal principal) {
```